core/iwasm/aot/arch/aot_reloc_dummy.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Intel Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "aot_reloc.h"
+#include <stdlib.h>
```

Comment:
better remove <stdlib.h>. *bh_platform.h* in *aot_runtime.h* will do.

Reply:
removed